grails-app/controllers/com/miniasaaslw/controller/payment/PaymentController.groovy

```diff
@@ -0,0 +1,28 @@
+package com.miniasaaslw.controller.payment
+
+import com.miniasaaslw.utils.LoggedCustomer
+
+class PaymentController {
+
+    def paymentService
+
+    def index() {
+        def errors = flash.errors
+
+        if (errors) {
+            return [errors: errors]
+        }
+    }
+
+    def delete() {
+        Long id = params.long("id")
+
+        try {
+            paymentService.delete(LoggedCustomer.CUSTOMER, id)
```

Comment:
No momento não tem cliente logado, se preocupem com isso quando a autenticação for desenvolvida. Se o código não tem utilidade agora, ele não precisa existir agora, podem por enquanto receber o Customer via parâmetro sem problemas e quando a autenticação for desenvolvida, ai sim será necessário alterar esse ponto.